inst/templates/ad_adeg.R

```diff
@@ -125,7 +125,9 @@ adeg <- adeg %>%
     AVALC = EGSTRESC
   ) %>%
 
-  # Derive new parameters based on existing records.
+  # Derive new parameters based on existing records. Note that, for the following
+  # four derive_param_* functions, only the variables specified in `by_vars` will
+  # be populated in the newly created records.
```

Comment:
Please put `derive_param_*()` in backticks add add a set of trailing parentheses.